File: C&C++/Saksham_Maggu/day_1.cpp
// ques 1: Write a program that prompts the user to enter their name and prints "HELLO, <Name>!".
#include<iostream>
#include<string>
using namespace std;
int main() 
{
    string s;
    cout<<"Enter name:"<<endl;
    cin>>s;
    cout<<"HELLO, "<<s<<"!"<<endl;
}

//-----------------------------------------------------------------------------------------------------

// ques 2: Write a program to calculate the sum of the digits of a three-digit number entered by the user.
#include<iostream>
using namespace std;
int sum_of_digits(int n)
{
    int digit_sum = 0;
    int temp = n;
    while (temp != 0) 
    {
        int digit = temp % 10;
        digit_sum += digit;
        temp /= 10;
    }
    return digit_sum;
}
int main() 
{
    int n;
    cout<<"Enter a 3 digit number:"<<endl;
    cin>>n;
    int sum=sum_of_digits(n);
    cout<<"Sum of digits: "<<sum<<endl;
}

File: Python/Harsh1/day1.py
#Write a Python program that prompts the user to enter their name and then prints a greeting message in the format "Hello, {Your Name}!".

#sol-
name=input('Enter the name:')
print(f'Hello, {name}')

#Write a Python program that takes a sentence as input and prints the reverse of each word in the sentence. For example, if the input is "Hello World!", the output should be "olleH dlroW!".

#sol-
sentence=input('Enter the sentence:')
length=len(sentence)
for i in range(1,length+1):
    print(sentence[-i], end='')
